dev/socket_port_test.py

```diff
@@ -0,0 +1,7 @@
+import importlib
+
+pysweepme = importlib.import_module("pysweepme", r"C:\Users\Ivan Ramirez\Documents\pysweepme")
+import pysweepme
+
+waferproberpath = r"C:\Users\Ivan Ramirez\Documents\SweepMe!\Devices\WaferProber-MPI_SENTIO"
+pysweepme.get_device("WaferProber-MPI_SENTIO", waferproberpath, "127.0.0.1:65432")
```

Comment:
shall this file be part of the PR? If so maybe adjust the paths to be more generic?